base/src/main/java/com/smartdevicelink/managers/screen/SoftButtonObject.java

```diff
@@ -295,12 +295,14 @@ public int getButtonId() {
     }
 
     /**
+     * DO NOT USE! let the managers assign ID's
      * Sets the id of the SoftButtonObject <br>
      * <strong>Note: If the developer did not set buttonId, the manager will automatically assign an id before the SoftButtons are sent to the head unit.
      * Please note that the manager may reuse ids from previous batch of SoftButtons that were already sent to the head unit</strong>
      *
      * @param buttonId an int value that represents the id of the SoftButtonObject
      */
+    @Deprecated
```

Comment:
Since this is still to be used by the BaseScreenManager should we make a note to make sure we change this to RestrictTo Library,  otherwise it might get removed when we clean up at the end of a future release